apps/editing-toolkit/editing-toolkit-plugin/site-editor/index.php

```diff
@@ -86,5 +61,5 @@ function is_site_editor_active() {
 	 *
 	 * @param bool true if Site Editor should be enabled, false otherwise.
 	 */
-	return apply_filters( 'a8c_enable_core_site_editor', false );
+	return is_readable( get_stylesheet_directory() . '/block-templates/index.html' );
```

Comment:
Same as above. I assumed it should be possible to remove this file completely? Unless there is some Dotcom weirdness going on. :)

Reply:
This function is still being used for:
1. Ensuring the above menu button is added in wp-admin.
2. The site API calypso uses to determine whether or not the 'site editor' button should be available from that end.

For the time being I figured this would be a good first step to allow us to start cleaning up our  blog sticker goo?

Reply:
I thought that 1 should work out of the box, but I see that that won't be the case unless we make some changes to core. If that worked we could rewrite the function calls in API and add this check inline (or `gutenberg_is_fse_theme` if it works).

But since 1 is not an option still, I think this is fine as a first step.

Reply:
> I thought that 1 should work out of the box

Me too!  I was a bit confused why it wasn't when I tried it that way.

> but I see that that won't be the case unless we make some changes to core.

Yeah, that may be an option.  Although Im not sure how much it would make sense for gutenberg to add a menu item that isnt listed as 'gutenberg'.  Alternatively we could try to update how dotcom handles hiding the gutenberg menus? 🤔 

> If that worked we could rewrite the function calls in API and add this check inline

Right!  And get rid of this file altogether? 😁   That would be nice.